js/app.js

```diff
@@ -1,55 +1,67 @@
-/* eslint-disable linebreak-style */
-/* eslint-disable no-use-before-define */
-/* eslint-disable no-unused-vars */
-/* eslint-disable-next-line quotes */
+/* eslint-disable import/extensions */
```

Comment:
- [ ] Good job refactoring and matching the wireframe design :clap:  However, please notice its not a good practice to disable the linter anywhere in our project, so please remove it and fix the issues raised by eslint.